### DISCUSSION TITLE
Using different thermistor is suitable?

### DISCUSSION THREAD

**Discussion by thanatas** [Jyers/Marlin#592] [Q&A] [answered]
I try to understand this thermistor will work on this firmware or not?

Or I need to compile new? The info I have had is following;

**100K thermistor - ATC Semitec 104GT-2/104NT-4-R025H42G(Used in ParCan & J-Head) (4.7k pullup)
Marlin -> #define TEMP_SENSOR_0 5**

As far as I understood, default one was write down "TEMP_SENSOR_0 1" in jyers's config.  So what is your suggestions?

I am currently using 5x5 HS, unfortunately I could not find its template in the firmware. Little help will be appreciated!

PC: Is it true way that Just changing current marlin config* file with jyers' template will work?

I am newbie for marlin build..

Thanks.

**Comment by tome9111991** [comment, marked as answer]
You can use BLTouch-3x3-HighSpeed config

  #define GRID_MAX_POINTS_X 3
And change grid to 5

**Reply by thanatas** [reply]
Yes, I figured out, thanks alot. 

I replace conf files with jyers. Is there anything to do before build it, The screen or anything ?

**Reply by tome9111991** [reply]
Should be work well

If somerhing wrong u can easy change config and build again

**Reply by thanatas** [reply]
Ok first, I must install volcano with new thermistor then I will try. 

Thanks a lot. I love the community!
